homeassistant/components/unifi/switch.py

```diff
@@ -232,12 +244,26 @@ class UnifiSwitchEntityDescription(
         icon="mdi:upload-network",
         api_handler_fn=lambda api: api.port_forwarding,
         control_fn=async_port_forward_control_fn,
-        device_info_fn=async_port_forward_device_info_fn,
+        device_info_fn=async_unifi_network_device_info_fn,
         is_on_fn=lambda hub, port_forward: port_forward.enabled,
         name_fn=lambda port_forward: f"{port_forward.name}",
         object_fn=lambda api, obj_id: api.port_forwarding[obj_id],
         unique_id_fn=lambda hub, obj_id: f"port_forward-{obj_id}",
     ),
+    UnifiSwitchEntityDescription[TrafficRules, TrafficRule](
+        key="Traffic rule control",
+        device_class=SwitchDeviceClass.SWITCH,
+        entity_category=EntityCategory.CONFIG,
+        icon="mdi:security-network",
+        api_handler_fn=lambda api: api.traffic_rules,
+        available_fn=lambda hub, obj_id: hub.available,
+        control_fn=async_traffic_rule_control_fn,
+        device_info_fn=async_unifi_network_device_info_fn,
+        is_on_fn=lambda hub, traffic_rule: traffic_rule.enabled,
+        name_fn=lambda traffic_rule: f"{traffic_rule.description}",
```

Comment:
```suggestion
        name_fn=lambda traffic_rule: traffic_rule.description,
```
description is already a string